pkg/cloud/ibmcloud/actuators/cluster/actuator.go

```diff
@@ -43,13 +41,17 @@ func NewActuator(params ActuatorParams) (*Actuator, error) {
 
 // Reconcile reconciles a cluster and is invoked by the Cluster Controller
 func (a *Actuator) Reconcile(cluster *clusterv1.Cluster) error {
-	log.Printf("Reconciling cluster %v.", cluster.Name)
-	// "TODO: Not yet implemented"
+	klog.Infof("Reconciling cluster %v.", cluster.Name)
+
+	klog.Infof("Reconciling cluster %v: Not implemented yet.", cluster.Name)
```

Comment:
I assume you have those 2 on purpose ,right?

Reply:
I will change it to `TODO` for second message